### ISSUE TITLE
Longhorn node id 

### ISSUE THREAD

**Issue by johansmitsnl** [rancher/k3s#960]
**Version:** v0.10.0 (f9888ca3)

**Describe the bug**
longhorn/longhorn#434         unable to mount a longhorn volume.

**To Reproduce**
* Follow the tutorial of Longhorn

**Expected behavior**
That the volume is mounted

**Actual behavior**
Can't mount

**Additional context**
I spoke with @yasker on [slack](https://cloud-native.slack.com/archives/CNVPEL9U3/p1571958670007100) and he asked me to file it here at k3s because the v.0.10.0 should have broke or revealed the issue again.

**Comment by ShylajaDevadiga** [comment]
@johansmitsnl We will have this fix in the next release. Temporary fix is to update the longhorn.yaml and set the default kubelet dir path as below
```
          - --kubelet-root-dir
          - /var/lib/kubelet
```
```
 k  get pv -A
NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                      STORAGECLASS   REASON   AGE
pvc-a9454e29-7e18-424f-8493-9507e3602e59   2Gi        RWO            Delete           Bound    default/longhorn-vol-pvc   longhorn                10s
k  get pvc -A
NAMESPACE   NAME               STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
default     longhorn-vol-pvc   Bound    pvc-a9454e29-7e18-424f-8493-9507e3602e59   2Gi        RWO            longhorn       23s
```

**Comment by johansmitsnl** [comment]
@ShylajaDevadiga yes this worked for me.

**Comment by ShylajaDevadiga** [comment]
Thanks for validating @johansmitsnl 
Closing issue.

**Comment by ShylajaDevadiga** [comment]
@johansmitsnl Le

**Comment by shuo-wu** [comment]
This issue is fixed in Longhorn v0.7.0+. See [this doc](https://github.com/longhorn/longhorn/blob/v0.7.0/docs/csi-config.md#k3s) for details.

**Comment by ShylajaDevadiga** [comment]
Thank yoy @shuo-wu . Closing issue.
